core/src/main/java/org/elasticsearch/cluster/node/DiscoveryNodes.java

```diff
@@ -163,6 +163,18 @@ public boolean nodeExists(String nodeId) {
     }
 
     /**
+     * Determine if a given node exists
+     *
+     * @param node of the node which existence should be verified
+     * @return <code>true</code> if the node exists. Otherwise <code>false</code>
+     */
+    public boolean nodeExists(DiscoveryNode node) {
```

Comment:
I see two usages of `nodeExists(String nodeId)` that could use this one instead. (LocalDiscovery and IndicesClusterStateServiceRandomUpdatesTests)